src/android/MsgHandle.java

```diff
@@ -222,7 +222,7 @@ public void handleMessage(Message msg) {
                     break;
                 }
 
-                final PluginResult wrongOSResult = new PluginResult(PluginResult.Status.ERROR, "Android OS is not supported");
+                final PluginResult wrongOSResult = pluginErrorResult("wrongOS", 1, "Android OS is not supported");
```

Comment:
does this mean the android *version* is not supported? obviously android operating system is supported, haha

Reply:
I believe this error is that a specific version of the `Android` OS is not supported.  It's an error that can be returned from the Library itself, so I just made it forward the error in this case.